src/_includes/accessibility/accessibility-manual.html

```diff
@@ -79,12 +79,12 @@ <h4 class="margin-top-xl text-muted">If an icon is <strong>not</strong> an inter
 
   <h4 class="margin-top-xl text-muted">If an icon represents an interactive element</h4>
   <p>
-    In the case of focusable interactive elements, there are various options to include an alternative text or label to the element, without the need for any visually hidden <code>&lt;span&gt;</code> or similar. For instance, simply adding the <code>title</code> attribute to the interactive element itself will be sufficient to provide an accessible alternative name for the element, as well as providing the mouse tooltip.
+    In the case of focusable interactive elements, there are various options to include an alternative text or label to the element, without the need for any visually hidden <code>&lt;span&gt;</code> or similar. For instance, simply adding the <code>aria-label</code> attribute with a text description to the interactive element itself will be sufficient to provide an accessible alternative name for the element, as well as providing the mouse tooltip.
```

Comment:
@talbs When using the aria-label approach this does not create a tooltip that you would get using the title attribute and so it might be better to encourage to create their own tooltips that reveal on focus of the element as well as on focus as well which is a limitation of the title attribute I believe but would benefit keyboard only users to be able to see a text based description using a custom tooltip